net.christianbeier.Gromit-MPX.yml

```diff
@@ -38,8 +38,8 @@ modules:
       - /share/dbus-1
     sources:
       - type: archive
-        url: https://download.gnome.org/sources/dconf/0.38/dconf-0.38.0.tar.xz
-        sha256: 45f60f41330d27715cce1315af123f94f1c2cdedb68b6bed3b309866eec44f58
+        url: https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
```

Comment:
Does this still apply and work?

Reply:
The patch? It applies cleanly, you can check the logs https://buildbot.flathub.org/#/builders/39/builds/11534/steps/6/logs/stdio line 5571